front/app/components/ProjectAndFolderCards/components/getCardSizes.ts

```diff
@@ -1,54 +1,37 @@
 import { TCardSize } from './PublicationStatusTabs';
-import { viewportWidths } from 'utils/styleUtils';
 
 export default function getCardSizes(
   adminPublicationsLength: number,
-  windowWidth: number
+  isLargerThanTablet: boolean
```

Comment:
This should be a nice perf improvement, a lot less rerenders of project/folder cards.

Reply:
(useWindowWidth was causing the many rerenders we noticed yesterday with getBreakpoint)